zingo-testutils/src/regtest.rs

```diff
@@ -175,11 +175,22 @@ pub fn launch_lightwalletd(
     }
     lightwalletd_child
 }
+fn write_zcash_conf(location: &PathBuf) {
+    // This is the only data we need to supply *to* the zcashd, the other files are created by zcashd and lightwalletd
+    use std::io::Write;
+    let conf_bytes: &'static [u8] = include_bytes!("../regtest/conf/zcash.conf");
+    File::create(location)
+        .unwrap()
+        .write_all(conf_bytes)
+        .unwrap();
+}
```

Comment:
this is definately an improvement and i'm not requesting changes here, but it would be nice for zingo-cli to read all the the regtest files from zingo-testutils instead of copying them eventually

Reply:
Since `zingo-cli` is a different crate that just depends on `zingo-testutils` it doesn't have access to data files that are part of `zingo-testutils`.

Probably a more elegant solution is to represent file contents as `const FILECONTENTS: &str = ..` inside of `zingo-testutils`.

Reply:
I vote we merge this as is.

Reply:
Sure. As I said im not requesting changes here.